enhancements/agent-installer/host-based-mirror.md

```diff
@@ -0,0 +1,282 @@
+---
+title: host-based-mirror
+authors:
+  - "@bfournie"
+reviewers:
+  - "@andfasano"
+  - "@patrickdillon"
+  - "@romfreiman"
+  - "@rwsu"
+approvers:
+  - "@zaneb"
+api-approvers:
+  - None
+creation-date: 2023-02-03
+last-updated: 2023-02-16
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/AGENT-262
+---
+
+# Host Based Mirror
+
+## Summary
+
+The agent based installer should have the ability to include all container
+images needed to boot a host in a disconnected environment, along with
+necessary tools, when it generates an ISO. This will allow the hosts to boot
```

Comment:
Just a thought: this proposal is very ISO-oriented. But there are relevant and timely use cases where it could be just as easy, or easier, to write container images to the physical disk at a factory or staging facility, rather than need to put them in an ISO. Maybe it makes sense to handle that scenario separately; but if the ISO management part of this is causing a lot of complication, it may be worth considering if a better starting point would be to just expect that the images are made available on some local volume, without facilitating exactly how.